packages/tools/pluggable-widgets-tools/scripts/e2e.js

```diff
@@ -142,9 +143,14 @@ async function getMendixVersion() {
         const dockerTagsResponse = await fetch(
             "https://registry.hub.docker.com/v1/repositories/mendix/runtime-base/tags"
         );
-        const runtimeVersions = (await dockerTagsResponse.json())
-            .filter(r => !r.name.startsWith("9"))
-            .map(r => r.name.split("-")[0]);
+
+        let dockerTagsJson = await dockerTagsResponse.json();
+
+        if (mendixMajorVersion) {
+            dockerTagsJson = dockerTagsJson.filter(r => r.name.startsWith(`${mendixMajorVersion}`));
```

Comment:
Template string is not really necessary here as the process env arguments are always string.

Reply:
I agree. I will remove it.